L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp2.cc

```diff
@@ -442,7 +555,7 @@ GlobalPoint HGCalTriggerGeometryV9Imp2::getTriggerCellPosition(const unsigned tr
                                .basicVector();
     }
   }
-  return GlobalPoint(triggerCellVector / cell_ids.size());
+  return GlobalPoint(triggerCellVector / static_cast<int>(cell_ids.size()));
```

Comment:
I'm not sure to see why a cast to int is needed

Reply:
i.e. this casting is  also done here HFNoseTriggerDetId nose_id(module_triggercells.first);

Reply:
Still, I don't understand why something which was working has been changed here. Besides, a vector size is unsigned so I don't see the logic of casting it to a signed integer. 
If there is valid reason for this change, I'm ok with that (but I'd like to know it); if there is no valid reason, please keep this code (which is independent from the HFNose updates) unchanged.